sw/device/rom_exts/rom_ext_manifest_parser.c

```diff
@@ -51,8 +51,17 @@ bool rom_ext_get_signature(rom_ext_manifest_t params,
   return true;
 }
 
-uint32_t rom_ext_get_image_len(rom_ext_manifest_t params) {
-  return mmio_region_read32(params.base_addr, ROM_EXT_IMAGE_LENGTH_OFFSET);
+rom_ext_extents_t rom_ext_get_extents(rom_ext_manifest_t params) {
```

Comment:
I am not sure about the naming `rom_ext_get_extents`. What about `...image_info, ...image_areas, ...image_ranges` or something similar. I find `extent` a bit confusing.

Reply:
I think `ranges` is a good name. While I don't have a huge problem with `extents`, I like "ranges" a little better.

Reply:
It's now `rom_ext_ranges_t` and `rom_ext_get_ranges` unless I typod somewhere.